sunpy/util/cond_dispatch.py

```diff
@@ -78,7 +78,7 @@
 
 from itertools import chain, repeat
 
-from sunpy.extern.six.moves import zip as izip
+from sunpy.extern.six.moves import zip
```

Comment:
@nabobalis here. Rather than letting modernize do it's thing, if we always do `from six.moves import zip` and relace any occurences of `izip` in the code with `zip. get me?